kafka/modules/kafka/2.6.0/install.sh

```diff
@@ -14,7 +14,7 @@ mv ${TMP}/* ${KAFKA_HOME}/
 cp -r ${KAFKA_HOME}/bin/kafka_exporter ${KAFKA_EXPORTER_HOME}/
 
 mkdir -p ${PRODUCT_LICENSE_DIR}/kafka
-cp ${KAFKA_HOME}/docs/licen*/* ${PRODUCT_LICENSE_DIR}/kafka
+cp -r ${KAFKA_HOME}/licen*/* ${PRODUCT_LICENSE_DIR}/kafka
```

Comment:
As discussed earlier, the Kafka 2.6.0 licenses directory has a different format from previous versions. The licenses are now nested within their own directories requiring `cp -r` to recursively copy them. In addition, the licenses are not held in the `docs` directory anymore